base/remote/R/remote.copy.to.R

```diff
@@ -22,6 +22,9 @@ remote.copy.to <- function(host, src, dst, options = NULL, delete = FALSE, stder
   if (as.logical(delete)) {
     args <- c(args, "--delete")
   }
+  if (is.null(host)) {
+    PEcAn.logger::logger.severe("Host param provided is NULL : Possibly a typo in the host name variable or a missing host in the arguments passed")
+  }
```

Comment:
Another approach that might catch more problems is to assert that `host` is a character value of length 1
```suggestion
 if(!is.character(host) | length(host) > 1)
    PEcAn.logger::logger.severe("`host` should be a length 1 character vector, not ", class(host), " length ", length(host))
  }
```